README.md

```diff
@@ -43,9 +43,11 @@ result of running the Multicore OCaml benchmarks.
 
 The currently included systems are:
 
-+ `eff` - [Eff](https://github.com/matijapretnar/eff)
-+ `hia` - [Handlers in Action](https://github.com/slindley/effect-handlers)
-+ `ocaml` - [Multicore OCaml](https://github.com/ocaml-multicore/ocaml-multicore)
++ `eff` - [Eff](https://github.com/matijapretnar/eff) [![Eff](https://github.com/effect-handlers/effect-handlers-bench/actions/workflows/system_eff.yml/badge.svg)](https://github.com/effect-handlers/effect-handlers-bench/actions/workflows/system_eff.yml)
++ `hia` - [Handlers in Action](https://github.com/slindley/effect-handlers) [![Handlers in Action](https://github.com/effect-handlers/effect-handlers-bench/actions/workflows/system_hia.yml/badge.svg)](https://github.com/effect-handlers/effect-handlers-bench/actions/workflows/system_hia.yml)
++ `koka` - [Koka](https://github.com/koka-lang/koka) [![Koka](https://github.com/effect-handlers/effect-handlers-bench/actions/workflows/system_koka.yml/badge.svg)](https://github.com/effect-handlers/effect-handlers-bench/actions/workflows/system_koka.yml)
++ `links` - [Links](https://github.com/links-lang/links) [![Links](https://github.com/effect-handlers/effect-handlers-bench/actions/workflows/system_links.yml/badge.svg)](https://github.com/effect-handlers/effect-handlers-bench/actions/workflows/system_links.yml)
++ `ocaml` - [Multicore OCaml](https://github.com/ocaml-multicore/ocaml-multicore) [![Multicore OCaml](https://github.com/effect-handlers/effect-handlers-bench/actions/workflows/system_ocaml.yml/badge.svg)](https://github.com/effect-handlers/effect-handlers-bench/actions/workflows/system_ocaml.yml)
```

Comment:
There is some redundancy here. The "passing badges" are displayed at the top of the README too. I think we should only display them once, either at the top or here.

Reply:
I'd go even further - if we make the benchmark table vertical, we could add an additional header row with the badges, plus make the system names clickable. Then, we could get rid of the whole Systems section and have less things to update when a new system appears.

Reply:
Would it make sense to inline benchmark descriptions in the table as well?

Reply:
I think that is a great idea.

Reply:
Inlining the description of benchmarks needs some fiddling to not enlarge the columns too much, but it still seems ok:
https://gist.github.com/jO-Osko/79832fab93698a7176eaccbc2717dbc8 (badges are broken, but this is irrelevant)

What do you think?

Reply:
This would also remove the `Benchmark` section

Reply:
Yes, this looks really nice and the `<br>`s are not that horrible.

Reply:
It looks good!

Reply:
It seems that gist renders it a bit differently, so double `<br>`s are not needed in descriptions as the lines wrap automatically. I also added another `<br>` after systems to align the badges

Reply:
Is there a good reason not to use the green check mark :heavy_check_mark:? The white check mark sort of blends with the white background on my monitors (I suppose this might be different if using dark mode).

Reply:
I guess this depends on the OS. I see
<img width="225" alt="Screenshot 2021-09-17 at 13 13 32" src="https://user-images.githubusercontent.com/1502128/133773659-16ffe614-a7e8-4006-b623-507552062476.png">
and
<img width="28" alt="Screenshot 2021-09-17 at 13 13 52" src="https://user-images.githubusercontent.com/1502128/133773684-9df328b3-46f7-4e24-835b-76560b1a97ff.png">

Reply:
Huh strange:
Dark macos:
![Screenshot 2021-09-17 at 13 15   ##21](https://user-images.githubusercontent.com/6787313/133774050-d8c76fbd-5f54-4b07-870b-a8ea56ff87c1.png)
Light windows:
![Screenshot 2021-09-17 at 13 18 21](https://user-images.githubusercontent.com/6787313/133774226-2d61aaac-5a71-4541-b325-f0af85a299a2.png)
![Screenshot 2021-09-17 at 13 18 08](https://user-images.githubusercontent.com/6787313/133774229-2733d3f2-92af-44dc-925d-4a7f7b559fec.png)

Reply:
Aha! Interesting. I do not see the green box on my end (Ubuntu 20.04 LTS).

Reply:
... and I see that the "green" check mark is rendered black/dark on your end.

Reply:
It does render differently: https://emojipedia.org/check-mark/, apparently this is a feature?
We can always default to 🟩  🟥

Reply:
No, that is not good for someone who is colour-blind. I wouldn't dwell too much on this issue because as you've said, there will be badges there at some point anyway.

Reply:
I agree with @matijapretnar, lets stick to the check marks, even though, they will be rendered differently on different platforms.